src/ucs/debug/debug.c

```diff
@@ -1019,6 +1019,12 @@ static void* ucs_debug_get_orig_func(const char *symbol, void *replacement)
     return func_ptr;
 }
 
+#if defined(HAVE_SIGHANDLER_T)
```

Comment:
empty `#if`?
maybe just `#if HAVE_SIGHANDLER_T` without definfed() ?
same for other places with use HAVE_xx